.hooks/prepare-commit-msg

```diff
@@ -14,6 +14,6 @@ ISSUE_NUMBER=$(echo $BRANCH_NAME | sed -n 's/.*-\([0-9]\)/\1/p')
 BRANCH_EXCLUDED=$(printf "%s\n" "${BRANCHES_TO_SKIP[@]}" | grep -c "^$BRANCH_NAME$")
 BRANCH_IN_COMMIT=$(grep -c "EPJ-$ISSUE_NUMBER" $1)
 
-if [ -n "$BRANCH_NAME" ] && ! [[ $BRANCH_EXCLUDED -eq 1 ]] && ! [[ $BRANCH_IN_COMMIT -ge 1 ]]; then
+if [ -n "$BRANCH_NAME" ] && [ -n "$ISSUE_NUMBER" ] && ! [[ $BRANCH_EXCLUDED -eq 1 ]] && ! [[ $BRANCH_IN_COMMIT -ge 1 ]]; then
   sed -i.bak -e "1s/^/EPJ-$ISSUE_NUMBER /" $1
```

Comment:
Issue Number should be added to the end of the message. Allowing the actual message to contain the important information at the beginning.